### PR TITLE
Tests / bugfix for unit power perks

### PR DESCRIPTION
**Pull request by internetfett**
Added tests for unit power perks.
Fixed a bug where a fourth parameter was expected for defense_from_land (typo in commit message)

### CODE REVIEW

**Comment by WaveHack** [comment]
Cool, thanks!

File: src/Calculators/Dominion/MilitaryCalculator.php [hunk 1]
```diff
@@ -376,7 +376,8 @@ protected function getUnitPowerFromLandBasedPerk(Dominion $dominion, Unit $unit,
         $landType = $landPerkData[0];
         $ratio = (int)$landPerkData[1];
         $max = (int)$landPerkData[2];
-        $constructedOnly = $landPerkData[3];
+        $constructedOnly = false;
+        //$constructedOnly = $landPerkData[3]; todo: implement for Nox?
```
Comment (WaveHack) [inline src/Calculators/Dominion/MilitaryCalculator.php]: Can probably be written as `$constructedOnly = $landPerkData[3] ?? false;`?

Comment (internetfett) [reply src/Calculators/Dominion/MilitaryCalculator.php]: Maybe with @$landPerkData[3]

Comment (WaveHack) [reply src/Calculators/Dominion/MilitaryCalculator.php]: Nooo, bad Fett! No error suppression tokens!! 😰

**Pull request merged**